package.json

```diff
@@ -36,7 +36,8 @@
     "react-dom": "^17.0.1",
     "readline": "^1.3.0",
     "sanitize-html": "^2.3.2",
-    "solc": "^0.6.6",
+    "solc0_8": "npm:solc@^0.8",
+    "solc0_6": "npm:solc@^0.6",
```

Comment:
@Tunji17 Can we also include compiler version 0.5.x and 0.7.x (where `x` is always the latest corresponding `PATCH` version). That way the node can support 4 versions out of the box.

Reply:
Yes i will include this